### ISSUE TITLE
Mqtt for cover

### ISSUE THREAD

**Issue by bubez81** [kiwi-cam/homebridge-broadlink-rm#201]
Hello, very useful plug-in, can I ask if there is the possibility to send the informations for covers over mqtt? Expecially the status in %?

**Comment by kiwi-cam** [comment]
Sorry, I'm not sure what you're asking.  The best documentation for our MQTT support is here: https://broadlink.kiwicam.nz/#mqtt-temperature-support.  If there's something you'd like to know that it doesn't explain please let us know.

**Comment by kiwi-cam** [comment]
So you'd like to retrieve the status of your window-covering via MQTT? Are you able to share the data/structure you get over MQTT and I'll look into it?

**Comment by bubez81** [comment]
Sorry, maybe I expressed myself badly, I'd like the plugin information to be displayed on mqtt as well, both the ability to read and send it.
For example:
if I use the plugin from homebridge and set the curtains to 50% I would like to be able to read the value also from mqtt (cover1 / position).
But if I send the information from mqtt (cover1 / setposition) the plugin sets the curtain to 50% based on the opening and closing time I have set on the plugin.
It could be possible?

In this way it would be possible to simultaneously use both homebridge (homekit / siri) and for example a second program, such as home-assistant
